packages/puppeteer-core/src/common/WaitTask.ts

```diff
@@ -225,6 +225,12 @@ export class WaitTask<T = unknown> {
         return;
       }
 
+      // Errors coming from WebDriver BiDi. TODO: Adjust messages after
+      // https://github.com/w3c/webdriver-bidi/issues/540 is resolved.
+      if (error.message.includes("Actor 'MessageHandlerFrame' destroyed")) {
```

Comment:
I would include the `AbortError: ` part here to really only let it apply for IPC issues when trying to send the response in Firefox.